src/main/java/bio/terra/cda/app/service/QueryService.java

```diff
@@ -46,7 +52,34 @@
   public QueryService(ObjectMapper objectMapper) {
     this.objectMapper = objectMapper;
   }
+  @CacheEvict
+  public void clearSystemStatus() {
+    logger.debug("Clear SystemStatus");
+  }
+
+
+  SystemStatus systemStatus = new SystemStatus();
+  @Cacheable
+  public SystemStatus bigQueryCheck(){
+    SystemStatusSystems bigQuerySystemStatus = new SystemStatusSystems();
+    logger.info("This was hit");
+    boolean success = false;
+    try{
+      String StatusCheck = bigQuery.getDataset("cda_mvp").getDatasetId().getDataset();
+      logger.info(StatusCheck);
+      success = StatusCheck.equals("cda_mvp");
+    }catch (Exception e){
+      logger.error("Status check failed ", e);
+    }
+    if(success){
+      bigQuerySystemStatus.ok(true).addMessagesItem("everything is fine");
+    }else{
 
+      bigQuerySystemStatus.ok(false).addMessagesItem("Error");
+    }
+    systemStatus.ok(bigQuerySystemStatus.getOk()).putSystemsItem("BigQuery",bigQuerySystemStatus);
```

Comment:
```suggestion
    systemStatus.ok(bigQuerySystemStatus.getOk()).putSystemsItem("BigQueryStatus",bigQuerySystemStatus);
```

Reply:
I don't know what this should be,  we are trying to indicate that the DataStore is reachable.  BigQuery:'ok' sounds like Google's not broken.   We may need this hook to marinate to determine what is the additional information needed in this return.

Reply:
That is build into the systemStatus class the ok method dose a boolen check and sets the value of ok